ts/index.ts

```diff
@@ -1467,17 +1467,17 @@ class Gossipsub extends Pubsub {
       const peers = getGossipPeers(this, topic, constants.GossipsubPrunePeers, (xid: string): boolean => {
         return xid !== id && this.score.score(xid) >= 0
       })
-      peers.forEach(p => {
+      for (const p of peers) {
```

Comment:
This path should use a `Promise.all` to allow multiple `getRawEnvelope` calls to be triggered concurrently.